src/App.js

```diff
@@ -1,11 +1,14 @@
+//Changed by Adam Copeland.
```

Comment:
you don't need to add your name here, as Git tracks every change, your name is attached to it as well. Please remove it